app/models/provider_interface/provider_relationship_permissions_form.rb

```diff
@@ -9,6 +9,8 @@ def assign_permissions_attributes(params)
     end
 
     def save!
+      @accredited_body_permissions.setup_at = Time.current
+      @training_provider_permissions.setup_at = Time.current
```

Comment:
Smol feature request: could we also have a `setup_by` recording the id of the provider user who has last changed org permissions? It would be very useful for support/auditing.

Alternatively, we should be adding to the audit trail. I realise this may be beyond the scope of this PR. But we should either add the `setup_by` or create a ticket to add these actions to the audit trail.

Reply:
I agree this should be using the audit trail, I was mistaken in thinking that using `save!` would do this but I need to use `update!(...)` - will amend.

Reply:
@mnacos https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2231/commits/13d1cf59d5ace13af84deeb81661cffd5c81a891 calls `update!` to include changes in audit trail.

Reply:
+1 to the audit trail. `save!` feels like it ought to work.

Reply:
@duncanjbrown I had a quick look at the audited gem and it seemed to work on `create` `update!` and `destroy`, I think it stores the parameters passed to the `create` and `update!` methods and uses them as details for the trail. https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/commit/13d1cf59d5ace13af84deeb81661cffd5c81a891 modifies the form to use `update!` so we should now record these permissions being changed.